src/renderer.ts

```diff
@@ -757,7 +764,7 @@ const renderEditions = (format: Format, excludeStyles = false) => (
 ): ReactNode => {
 	switch (element.kind) {
 		case ElementKind.Text:
-			return textRenderer(format, excludeStyles, element);
+			return textRenderer(format, excludeStyles, element, false);
```

Comment:
Just entertaining a thought here, maybe we should use an object syntax when the function arity is above 2 or 3? So it's easier to read the arguments. Not necessarily for this PR but speaking in general as a style guide. It's a shame JS (and in consequence TS) don't support named parameters like a lot of other languages.
What are your thoughts? @abeddow91 @joecowton1 @JamieB-gu

Reply:
Although with TypeScript and language tools it's easy to read the signature of the function .. so maybe not worth the cost of creating an object.